src/peft/peft_model.py

```diff
@@ -149,7 +155,7 @@ def active_adapters(self) -> list[str]:
             adapters = self.active_adapter
             if isinstance(adapters, str):
                 adapters = [adapters]
-        return adapters
+        return list(filter(lambda x: len(x) > 0, adapters))
```

Comment:
Not sure why this is needed. Is this because of `self.active_adapter = ""`. Could you please explain the reason behind that?